src/main/java/com/google/cloud/spanner/jdbc/JdbcParameterStore.java

```diff
@@ -814,13 +819,13 @@ private Builder setArrayValue(ValueBinder<Builder> binder, int type, Object valu
     } else if (Long[].class.isAssignableFrom(value.getClass())) {
       return binder.toInt64Array(toLongList((Long[]) value));
     } else if (float[].class.isAssignableFrom(value.getClass())) {
-      double[] l = new double[((float[]) value).length];
+      float[] l = new float[((float[]) value).length];
```

Comment:
Is this code still needed to handle some contexts?

Why not use `binder.toFloat32Array((float[]) value);` now?

Reply:
Good point, that is no longer necessary. Thanks for catching it.